roles/shiftstack/tasks/post_deploy_shiftstackclient_pod.yml

```diff
@@ -0,0 +1,7 @@
+- name: "Copy the RHOSO's management kubeconfig to '{{ cifmw_shiftstack_client_pod_name }}'"
```

Comment:
Can you please add the common header that is being added across the ci-framework files?
```suggestion
---
# Copyright Red Hat, Inc.
# All Rights Reserved.
#
# Licensed under the Apache License, Version 2.0 (the "License"); you may
# not use this file except in compliance with the License. You may obtain
# a copy of the License at
#
#     http://www.apache.org/licenses/LICENSE-2.0
#
# Unless required by applicable law or agreed to in writing, software
# distributed under the License is distributed on an "AS IS" BASIS, WITHOUT
# WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied. See the
# License for the specific language governing permissions and limitations
# under the License.

- name: "Copy the RHOSO's management kubeconfig to '{{ cifmw_shiftstack_client_pod_name }}'"
```

Reply:
Done